src/Symfony/Component/Security/Http/Attribute/IsCsrfTokenValid.php

```diff
@@ -11,14 +11,16 @@
 
 namespace Symfony\Component\Security\Http\Attribute;
 
+use Symfony\Component\ExpressionLanguage\Expression;
+
 #[\Attribute(\Attribute::IS_REPEATABLE | \Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD | \Attribute::TARGET_FUNCTION)]
 final class IsCsrfTokenValid
 {
     public function __construct(
         /**
          * Sets the id used when generating the token.
```

Comment:
The docblock is not valid anymore

Reply:
updated with a proposition, any other proposition is welcomed